### PR TITLE
Updated centos-8 image + added README status badges for test status

### PR DESCRIPTION
**Pull request by AhmedBM**
## Description

* Updated centos-8 to also include omi
* Updated vscode settings to default to jsonc (JSON with comments)
* Updated README to include test badges for CI + Integration Test workflow status

## Checklist

- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-osconfig/blob/main/CONTRIBUTING.md).
- [X] All unit tests are passing.
- [X] I have merged the latest `main` branch prior to this PR submission.
- [x] I submitted this PR against the `main` branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # OSConfig
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](LICENSE.md)
+[![CI Tests](https://github.com/Azure/azure-osconfig/actions/workflows/ci.yml/badge.svg)](https://github.com/Azure/azure-osconfig/actions/workflows/ci.yml)
+[![Integration - NRP Tests](https://github.com/Azure/azure-osconfig/actions/workflows/universalnrp-test.yml/badge.svg)](https://github.com/Azure/azure-osconfig/actions/workflows/universalnrp-test.yml)
```
Comment (MariusNi) [inline README.md:5]: What are these doing here? Please remove

**Review by MariusNi — CHANGES_REQUESTED** [review]
:clock1:

**Review by robertschaedler3 — DISMISSED** [review]

File: devops/README.md [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+
 # Package Installation
 
 Our packages are distributed via the [Microsoft repository](https://packages.microsoft.com/). The following instructions will guide you through the installation of the `osconfig` package on our supported Linux distributions. Additionally, you can find our MSDN documentation on package installation at [docs.microsoft.com](https://learn.microsoft.com/azure/osconfig/howto-install?tabs=package).
 
 Channels | Description
 -----|-----  
-[![insiders-fast](https://github.com/Azure/azure-osconfig/actions/workflows/insiders-fast.yml/badge.svg)](https://github.com/Azure/azure-osconfig/actions/workflows/insiders-fast.yml) | The latest and greatest features and bug fixes. This channel is updated frequently.
-[![prod](https://github.com/Azure/azure-osconfig/actions/workflows/prod.yml/badge.svg)](https://github.com/Azure/azure-osconfig/actions/workflows/prod.yml) | The stable channel. This channel is updated less frequently and is recommended for production use.
+`insiders-fast` | The latest and greatest features and bug fixes. This channel is updated frequently.
```
Comment (MariusNi) [inline devops/README.md:12]: >he latest and greatest features and bug fixes. This channel is updated frequently. [](http://example.com/codeflow?start=19&length=82)

May want to add here that insiders-fast packages are not guaranteed to work (no promise that they were validated by us), only prod ones are.

**Review by MariusNi — DISMISSED** [review]
:shipit:

**Review by robertschaedler3 — APPROVED** [review]

File: devops/README.md (hunk 2 above)
Comment (AhmedBM) [reply devops/README.md]: Reworded

**Review by MariusNi — APPROVED** [review]
:shipit:

**Pull request merged**